l1-contracts/contracts/bridge/L1SharedBridge.sol

```diff
@@ -82,12 +82,13 @@ contract L1SharedBridge is IL1SharedBridge, ReentrancyGuard, Ownable2StepUpgrade
 
     /// @dev Indicates whether the hyperbridging is enabled for a given chain.
     // slither-disable-next-line uninitialized-state
-    mapping(uint256 chainId => bool enabled) internal hyperbridgingEnabled;
+    mapping(uint256 chainId => bool enabled) public hyperbridgingEnabled;
 
-    /// @dev Maps token balances for each chain to prevent unauthorized spending across hyperchains.
-    /// This serves as a security measure until hyperbridging is implemented.
-    /// NOTE: this function may be removed in the future, don't rely on it!
-    mapping(uint256 chainId => mapping(address l1Token => uint256 balance)) public chainBalance;
+    /// @dev A mapping assetInfo => assetAddress
+    mapping(bytes32 _assetInfo => address _assetAddress) public assetAddress;
```

Comment:
asset vault address

Reply:
+

Reply:
Maybe AssetHandler instead ( it is not always a vault, sometimes it is the asset itself)